dashboard/src/components/Header/ContextSelector.tsx

```diff
@@ -165,6 +166,7 @@ function ContextSelector() {
                 className="clr-page-size-select"
                 onChange={selectNamespace}
                 value={namespace}
+                data-testid="select-namespace"
```

Comment:
Yeah, this was one that I really struggled to find a way to reference it by role or label in our tests, without a test-id (I didn't want to restructure the markup as part of the change). I just took another look and found I can add an `aria-label` attribute to the `select` for cluster and namespace, which is better than adding a test-id (since it'll actually function with a correct label text, accessibility-wise). We could also update the `span` elements to be `label`s but I didn't want to update the markup and, it seems, was being lazy, then I thought you're going to review this too and it's what you'd do, so I've done that instead... good reason for having reviewers. Pushing to a different PR.

Reply:
Thanks for the explanation and follow-up PR. Happy to be back again!